tests/ttnn/unit_tests/operations/eltwise/test_add.py

```diff
@@ -34,7 +40,7 @@ def test_non_4D_channel_bcast(device, shapes):
 
 
 @pytest.mark.parametrize("scalar", [3])
-@pytest.mark.parametrize("size", [64])
+@pytest.mark.parametrize("size", [64, 1, 0])
```

Comment:
Can we have a similar test for 1D tensor and 0D tensor (scalar) or is it out of scope of this PR?

Reply:
This lines means to test with size [64], [1], and [0]. So it does test 1D tensors